src/scripts/inc/common.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
+TMP_DIR="/tmp"
+RUN_USER="alexapi"
+HOME_DIR="/var/lib/AlexaPi"
+
+function run_python {
+    /usr/bin/python2 $@
+}
+
+function run_pip {
```

Comment:
Only "install" pip command is used. It would it be cleaner to rename `run_pip` to `pip_install` and change line 12 to:
```
$(which pip2) install $@
```
On my RPI2 and RPI3 pip2 is in `/usr/local/bin`, so it is better to not hardcode `/usr/bin`